### ISSUE TITLE
Deployments report not showing up (Truffle migrations)

### ISSUE THREAD

**Issue by sarveshwar-singh** [cgewecke/eth-gas-reporter#238]
truffle-config.js:
```
  mocha: {
    reporter: 'eth-gas-reporter',
    reporterOptions : {
      'currency': 'JPY'
    }
  },
```

truffle version:
```
Truffle v5.1.63 (core: 5.1.63)
Solidity - 0.7.6 (solc-js)
Node v14.15.4
Web3.js v1.2.9
```

I could see **Methods** report with correct values but **Deployments** reports does not shows up.
 
I am on 
`"eth-gas-reporter": "^0.2.20",`
`"ganache-cli": "^6.12.2",`

how could I fix this?
Thanks !!

**Comment by cgewecke** [comment]
Ah! Very sorry, this is a bug introduced in `0.2.19`. Thanks for reporting.

If you downgrade to `0.2.18` deployment data from migrations should show up in the report.

**Comment by sarveshwar-singh** [comment]
@cgewecke 
It works like charm on `0.2.18`.
Thanks for great work!!

where can i track this bug, so in order to upgrade to the latest version when fixed?

**Comment by cgewecke** [comment]
@sarveshwar-singh Oh good. 

I will close this issue when I publish the fix - you should get a notification from Github via email since you opened it.

**Comment by cgewecke** [comment]
Should be fixed in the latest version: `0.2.21`

Please just lmk if you're still seeing problems with this.

**Comment by sarveshwar-singh** [comment]
@cgewecke 
Yes its working on: `eth-gas-reporter@0.2.21`
Thanks !!
